lib/config.js

```diff
@@ -9,7 +9,8 @@ const coffee = require('coffee-script');
 const debug = require('debug')('brunch:config');
 const deepAssign = require('deep-assign');
 
-const loadInit = require('deppack').loadInit;
+const deppack = require('deppack'); // isNpm
```

Comment:
No, it's not about `this`. It's about the fact that most of deppack functions only become exposed after the call to `loadInit`: see https://github.com/brunch/deppack/blob/master/index.js